test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -420,6 +420,19 @@ class test_ucp_wireup_1sided : public test_ucp_wireup {
         }
     }
 
+   ucp_object_version_t address_version() {
+        ucp_object_version_t addr_version =
```

Comment:
IMO this function should return according to get_variant_value() and caller can do EXPECT_EQ
also can this function be const?